mkdocs/docs/getting-started/first-experiment.md

```diff
@@ -147,9 +212,13 @@ kubectl delete -n default -f $ITER8/samples/deployments/app/deploy.yaml
 
 **Next Steps**
 
-!!! tip "Use with your app"
-    1. Run the above experiment with your app by setting the `URL` value in the Helm command to the URL of your app. 
+!!! tip "Use in your environment"
+    1. Run the above experiment with your app by setting the `URL` value in the Helm command to the URL of your app. You can run this experiment in any Kubernetes environment such as a dev, test, staging, or production cluster.
```

Comment:
Nitpicking but I don't like the value of `URL`.

"By setting the `URL` in the Helm command..."

Reply:
Is running things in a production cluster something we want to explicitly suggest?

Reply:
Production suggestion is good to include, I think. At the minimum, for the sake of completeness.